webapp/sass/layout/_sidebar-left.scss

```diff
@@ -39,12 +39,14 @@
         }
     }
 
-    .dropdown-menu {
+    .navbar-nav > li > .dropdown-menu {
         max-height: 80vh;
         max-width: 200px;
         overflow-x: hidden;
         overflow-y: auto;
         width: 200px;
+        margin-top: 3px;
+        margin-left: 2px;
```

Comment:
@yuters I think this should be `margin-right` rather than `margin-left`, because this is what I see on the test server. Even if it `margin-right` the spacing would still not be equal so I suggest reducing the `width` to `199px` and a `margin-right: 3px;`

![screen shot 2016-07-28 at 6 24 54 pm](https://cloud.githubusercontent.com/assets/11034289/17214133/e9fc4f92-54f0-11e6-82b1-6aea3c4045bc.png)